pro/build-nextflow-schemas/action.yml

```diff
@@ -19,6 +19,10 @@ inputs:
       Release of Viash tools to use.
     required: false
     default: 'latest'
+  enable_dataset_input:
```

Comment:
Could you add the input type (and perhaps also to the other arguments)? Here is an example: https://github.com/openpipelines-bio/openpipeline/blob/e1895f1c70f820796581d5889bdd948aa18cb286/.github/workflows/main-build.yml#L9C7-L9C22